src/interfaces/messages/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { BaseMessageReply } from '../../core/message-reply.js';
+import type { BaseMessage, Descriptor } from '../../core/types.js';
+import type { DwnInterfaceName, DwnMethodName } from '../../core/message.js';
+
+export type MessagesGetDescriptor = Pick<Descriptor, 'interface' | 'method' > & {
+  interface : DwnInterfaceName.Messages;
+  method: DwnMethodName.Get;
+  messageCids: string[];
+};
+
+export type MessagesGetMessage = BaseMessage & {
+  descriptor: MessagesGetDescriptor;
+};
+
+// oof horrible name
+export type MessagesGetReplyEntry = {
```

Comment:
Another idea for a refactor: I've never used [typescript namespaces](https://www.typescriptlang.org/docs/handbook/namespaces.html#namespaced-validators) before, but it might make these type names less ridiculous.

Like so:
```
namespace MessagesGet {
  export type ReplyEntry = {
     ...
  };

  export type Message {
     ...
  }
}
```

Again this idea is outside the scope of this PR, so I'm fine with this goofy variable name for now.